### ISSUE TITLE
Empty block name error with module-price-permissions

### ISSUE THREAD

**Issue by nataliefox-288** [magento/magento2#36202]
### Preconditions and environment

Magento version: 2.4.5
There appears to be a bug in the enterprise module module-price-permissions when a block with no name is used (on an admin screen). A message is seen:
Deprecated Functionality: stripos(): Passing null to parameter #1 ($haystack) of type string is deprecated in /app/vendor/magento/module-price-permissions/Observer/AdminhtmlBlockHtmlBeforeObserver.php on line 125

### Steps to reproduce

Using the Advanced Feeds module.
Navigate to 
Catalog > Feeds

### Expected result

Should be able to see list of feeds that have been set up.

### Actual result

The error message is seen.

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [X] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-November** [comment]
Hi @nataliefox-288 ,
Thank you for reporting and collaboration. However, Advanced Feeds module is a 3rd party extension and code of this extension is not part of https://github.com/magento/magento2 git repository. We are closing this issue as community is not able to provide fix for it in this repository. All questions, issue reports and fix for them should be addressed to the corresponding extension owners(support) on the [Magento Market place](https://marketplace.magento.com/extensions.html) page
Thank you.

**Comment by maxlussier** [comment]
For those interested I have contacted Mirasvit Dev Team to inform them of the situation. They informed me that they will include the fix in their next release (1.2.9). In the meantime, here's the patch:

```
diff --git a/vendor/mirasvit/module-feed/src/Feed/Ui/Feed/Listing/DataProvider.php b/vendor/mirasvit/module-feed/src/Feed/Ui/Feed/Listing/DataProvider.php
--- a/vendor/mirasvit/module-feed/src/Feed/Ui/Feed/Listing/DataProvider.php
+++ b/vendor/mirasvit/module-feed/src/Feed/Ui/Feed/Listing/DataProvider.php
@@ -130,6 +130,7 @@
     {
         $statisticksBlock = $this->statisticksBlockFactory->create();
         $statisticksBlock->setData('feed', $item)
+            ->setNameInLayout('mst_feed_feed_statistics')
             ->setData('interval', $days);

         return $statisticksBlock->toHtml();

```
